src/main/java/com/google/sps/data/AttendeeInterface.java

```diff
@@ -0,0 +1,14 @@
+package com.google.sps.data;
+
+import java.util.Date;
+import com.google.appengine.api.datastore.Entity;
+
+/** Interface for the Attendee class. */
+interface AttendeeInterface{
```

Comment:
Each function (maybe minus the getters since they're the most obvious), should have its own comment explaining what it does